### PR TITLE
Add service id to service blocks

### PR DESCRIPTION
**Pull request by makyo**
As a driveby, this also removes dependencies.tsv, as this is a library, not a command.  A juju plugin will be created down the line to generate SVGs on the user's system which will utilize this library.

### CODE REVIEW

**Comment by makyo** [comment]
# QA

* tests should pass
* run `go run examples/generatesvg.go examples/charmworld.yaml` and ensure that the `<use>` tags for service blocks include an ID element.

**Comment by bac** [comment]
:+1: and QA- :ok: 

Just did the QA as described above.  Thanks for the branch and the drive-by.

File: dependencies.tsv [hunk 1]
```diff
@@ -1,16 +0,0 @@
-github.com/ajstarks/svgo	git	89e3ac64b5b3e403a5e7c35ea4f98d45db7b4518	2014-10-04T21:11:59Z
-github.com/juju/errors	git	036046bfdccf6f576e2e5dec7f7878597bcaebe7	2015-02-11T20:59:49Z
-github.com/juju/gojsonpointer	git	0154bf5a168b672d8c97d8dd83a54cb60cd088e8	2014-07-18T03:59:30Z
-github.com/juju/gojsonreference	git	f0d24ac5ee330baa21721cdff56d45e4ee42628e	2015-02-04T19:46:33Z
-github.com/juju/gojsonschema	git	e1ad140384f254c82f89450d9a7c8dd38a632838	2015-03-12T17:00:16Z
-github.com/juju/loggo	git	dc8e19f7c70a62a59c69c40f85b8df09ff20742c	2014-11-17T04:05:26Z
```
Comment (urosj) [inline dependencies.tsv]: No logs? I wonder if having logs would help, at least error logs. Maybe it's enough to have it just in the client.

File: dependencies.tsv [hunk 2]
```diff
@@ -1,16 +0,0 @@
...
-github.com/juju/names	git	ce4ecb2967822062fc606e733919c677c584ab7e	2015-02-20T07:57:36Z
-github.com/juju/schema	git	27a52be50766490a6fd3531865095fda6c0eeb6d	2014-07-23T04:23:18Z
-github.com/juju/testing	git	b93c38205ac2e3817b7b1d9d60e89e69c99a6362	2015-03-16T13:08:43Z
-github.com/juju/utils	git	a90aa2e02b9e7fe354ab816e05b1e0a77f27242d	2015-02-23T16:02:32Z
-golang.org/x/crypto	git	4ed45ec682102c643324fae5dff8dab085b6c300	2015-01-12T22:01:33Z
-gopkg.in/check.v1	git	64131543e7896d5bcc6bd5a76287eb75ea96c673	2014-10-24T13:38:53Z
-gopkg.in/errgo.v1	git	81357a83344ddd9f7772884874e5622c2a3da21c	2014-10-13T17:33:38Z
```
Comment (urosj) [inline dependencies.tsv]: No errors?

**Comment by urosj** [comment]
This PR LGTM, however no errors (and to some degree no logs as well) worries me a bit. Haven't looked a the complete code TBH, but I wonder if error handling can really be skipped.

**Comment by makyo** [comment]
@urosj That was simply removing dependencies.tsv - errgo is still imported in jujusvg.go

**Comment by makyo** [comment]
Thanks all

**Pull request merged**